packages/data-point-express/lib/middleware.js

```diff
@@ -42,20 +42,23 @@ function createErrorMessage(err) {
   };
 }
 
-function resolveReducer(dataPoint, reducer, options, res, initialValue = {}) {
-  dataPoint
-    .transform(reducer, initialValue, options)
-    .then(acc => {
-      if (typeof acc.value === "string") {
-        res.send(acc.value);
-        return;
-      }
-      res.json(acc.value);
-    })
-    .catch(err => {
-      res.status(400).send(createErrorMessage(err));
-    })
-    .done();
+async function resolveReducer(
```

Comment:
Is it possible to pass object with all these params?

Reply:
it is possible, so far in data-point, we have not adopted that pattern, I'm open to it when possible, especially in cases where we have too many arguments I see the advantage. 

In this case its outside the scope of this PR